src/services/auth/auth.types.ts

```diff
@@ -6,11 +6,6 @@ export type Network = (typeof network)[number];
 export const accountTypes = ['singleton', 'lite'] as const;
 export type AccountType = (typeof accountTypes)[number];
 
-export const ProofPayloadSchema = z.object({
```

Comment:
I think the proof payload should include the account like it is here. What if for some reason I use the same private key for multiple accounts? In that case, a bad actor can copy the signed message and login to my other account.

Also, the signed message of just a timestamp is very vague. It could literally mean anything. At least when you have an account added to the message, there is some verification of user intent.

Reply:
Yep, valid point, I didn't consider keys controlling several accounts. If an attacker has managed to access your signed message and authenticate himself within a minute though we have a big problem.